projects/Draupnir/default.nix

```
@@ -32,5 +32,10 @@
         tests.basic = "${sources.inputs.nixpkgs}/nixos/tests/matrix/draupnir.nix";
       };
     };
+    demo.vm = {
+      module = ./example.nix;
+      description = "Deployment for demo purposes";
+      tests.basic = "${sources.inputs.nixpkgs}/nixos/tests/matrix/draupnir.nix";
```

So we don't display the unfinished config (see https://github.com/ngi-nix/ngipkgs/issues/1139)

```suggestion
      tests.basic = "${sources.inputs.nixpkgs}/nixos/tests/matrix/draupnir.nix";
      problem.broken.reason = ''
        Still a work in progress.
      '';
```